pkg/packagekit/updates.jsx

```diff
@@ -56,18 +56,43 @@ const PK_STATUS_STRINGS = {
     14: _("Verifying"),
 }
 
+const I_TRANS = "org.freedesktop.PackageKit.Transaction";
+
 var dbus_pk = cockpit.dbus("org.freedesktop.PackageKit", {superuser: "try", "track": true});
 var packageSummaries = {};
 
-function pkTransaction() {
+function pkWatchTransaction(transObj, signalHandlers, notifyHandler) {
```

Comment:
Do you think `transactionObject` would be too long here? The other variables aren't abbreviated.